### PR TITLE
Let `convert_to_xlsx_using_vbscript` try 3 times, because VBScript is flaky.

### PR DESCRIPTION
**Pull request by rudolfbyker**

### CODE REVIEW

File: aa_py_xl_convert/_vbscript.py [hunk 1]
```diff
@@ -1,9 +1,11 @@
 from pathlib import Path
 from subprocess import run
 
+from retry import retry
 from locate import this_dir
 
 
+@retry(tries=3, delay=2, backoff=2)
```
Comment (malankriel) [inline aa_py_xl_convert/_vbscript.py:8]: Nice one! What backoff do? And how is a delay of 2?

Comment (malankriel) [reply aa_py_xl_convert/_vbscript.py:8]: Read the docs!! https://pypi.org/project/retry/

> @retry((ValueError, TypeError), delay=1, backoff=2)
def make_trouble():
    '''Retry on ValueError or TypeError, sleep 1, 2, 4, 8, ... seconds between attempts.'''

**Comment by malankriel** [comment]
Is 3 times enough? :)

**Comment by rudolfbyker** [comment]
No idea ... :)

**Pull request merged**